tools/halide_image_io.h

```diff
@@ -1192,7 +1192,7 @@ bool save_mat(ImageType &im, const std::string &filename) {
             check(false, "unreachable");
         };
         break;
-    case halide_type_handle:
+    default:
```

Comment:
Might be worth also adding the bfloat type case and giving an explicit error message that bfloat is not supported. (The default should remain in place of course.)